packages/graph/hash_graph/lib/graph/src/store/postgres/query/expression/conditional.rs

```diff
@@ -71,6 +72,7 @@ impl<'q> Expression<'q> {
 impl Transpile for Expression<'_> {
     fn transpile(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
         match self {
+            Self::Asterix => fmt.write_char('*'),
```

Comment:
Just a quick question about the transpilation here. How come we add spaces in front of `OVER` here and not for `*`?

Reply:
`Window` will print the `expression` before, so both have no space in front